src/styles/blocks/card__text.scss

```diff
@@ -0,0 +1,15 @@
+.card__text {
+  margin-bottom: 4px;
+  color: $main-text-color;
+  width: 166px;
+  height: 36px;
+  top: 207px;
+  left: 17px;
```

Comment:
The `top` and `left` properties will have no effect unless the element has a positioning context. Consider adding `position: relative;` or another positioning property if needed.